packages/playground/src/router/routes.ts

```diff
@@ -62,6 +62,7 @@ enum ApplicationRoutes {
   Discourse = "/deploy/labs/discourse/",
   Taiga = "/deploy/labs/taiga/",
   Owncloud = "/deploy/labs/owncloud/",
+  Openwebui = "/deploy/applications/openwebui/",
```

Comment:
Please update the route
```suggestion
  Openwebui = "/deploy/laps/openwebui/",
```